tutorials/shaders/shader_reference/particle_shader.rst

```diff
@@ -138,21 +114,65 @@ Process built-ins
 +------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------+
 | in uint **FLAG_EMIT_CUSTOM**       | A flag for using on the last argument of ``emit_subparticle`` function to assign a custom data vector to a new particle.                |
 +------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------+
-| in bool **COLLIDED**               | ``true`` when the particle has collided with a particle collider.                                                                       |
-+------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------+
-| in vec3 **COLLISION_NORMAL**       | A normal of the last collision. If there is no collision detected it is equal to ``vec3(0.0)``.                                         |
+| in vec3 **EMITTER_VELOCITY**       | Velocity of the Particles node.                                                                                                         |
 +------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------+
-| in float **COLLISION_DEPTH**       | A length of normal of the last collision. If there is no collision detected it is equal to ``0.0``.                                     |
+| in float **INTERPOLATE_TO_END**    | Value of [code]interp_to_end[/code] property of Particles node.                                                                         |
 +------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------+
-| in vec3 **ATTRACTOR_FORCE**        | A combined force of the attractors at the moment on that particle.                                                                      |
-+------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------+
-| in vec4 **USERDATAX**              | Vector that enables the integration of supplementary user-defined data into the particle process shader.                                |
-|                                    | ``USERDATAX`` are six built-ins identified by number, ``X`` can be numbers between 1 and 6.                                             |
+| in uint **AMOUNT_RATIO**           | Value of [code]amount_ratio[/code] property of Particles node.                                                                          |
```

Comment:
You're accidentally using the rst syntax here and below

Reply:
Ah, I am. I caught my mistake and fixed it in a few places, but I guess I missed updating it everywhere.

Reply:
Fixed! Thank you